web/src/pages/apd/activities/cost-allocation/CostAllocateFFPQuarterly.js

```diff
@@ -80,21 +80,17 @@ const CostAllocateFFPQuarterly = ({
     }
   }, [quarterlyFFP[year].subtotal.contractors]); // eslint-disable-line react-hooks/exhaustive-deps
 
-  const setInHouse =
-    quarter =>
-    ({ target: { value } }) => {
-      setValue(`[${quarter}].inHouse.percent`, value / 100);
-      setInHouseFFP(activityIndex, year, quarter, value);
-      announce(activityId, year, quarter, 'inHouse');
-    };
+  const setInHouse = quarter => value => {
```

Comment:
you could just combine these args and functions here and have
```
const setInHouse = (quarter, value) => {
...
```
then when you use it down below you could just call
```
setInHouse(q, value)
```
it was curried before so that you could use it like
```
onChange={setInHouse(q)}
```
and then the onChange would be called on the function above and you don't have to destructure the value. I can explain that better if that made no sense